.teamcity/_self/projects/WebApp.kt

```diff
@@ -733,7 +733,7 @@ object RunCalypsoPlaywrightE2eTests : BuildType({
 				export PLAYWRIGHT_BROWSERS_PATH=0
 
 				# Install modules
-				${_self.yarn_install_cmd}
+				yarn install
```

Comment:
I've had to change this to explicitly run `yarn install` since the pre-populated `node_modules` cache from [this change](https://github.com/Automattic/wp-calypso/commit/7efc3246ac99b30f24a6040fc4bfeaf3791f7d10#diff-c08774f332e4c025ce1da0c7036888d494b6d981ca4b073a4f2266ec55a6bfbc) does not include Playwright binaries.

Reply:
Any clue why that's the case? the directory should be a copy of the output from `yarn install` once daily. So it would be weird to me that the binaries aren't there even after a second yarn install, but running `yarn install` by itself does include them :thinking:

Reply:
> Any clue why that's the case?

I suspect it is because of the environment variable exported above:
`export PLAYWRIGHT_BROWSERS_PATH=0`
without this, it seems the Playwright binaries are not installed in the local `node_modules` directory.

I suppose I can try adding the export to [this file](https://github.com/Automattic/wp-calypso/blob/7efc3246ac99b30f24a6040fc4bfeaf3791f7d10/.teamcity/_self/yarnInstall.kt). My concern was that it would affect non-Playwright jobs.

Do you think that change is safe to make?

Reply:
Gotcha! In that case the current case makes sense to me. I think if `yarn` has an existing node_modules which is mostly up to date, it must not be trying to do the install step which handles that variable. Thanks for the explanation :+1:

Reply:
With that said, I should put a reminder somewhere to make that change at some point, so all E2E definitions use the same call. Just reduces possibility that something somewhere goes wrong. So thanks for pointing this out!